init.c

```diff
@@ -1905,26 +1909,35 @@ static void mutt_restore_default (struct option_t *p)
 #endif
 }
 
+static void ESC_char(char C, char* p, char *dst, size_t len)
```

Comment:
change the name of the function to all lower case, maybe `esc_char(…)` or `escape_char(…)` or `escaped_char(…)`